css/tailwind.css

```diff
@@ -4,25 +4,6 @@
 
 /* Global styles automatically moved into base layer */
 @layer base {
-  @font-face {
```

Comment:
This is done with that Inter class right?

Reply:
Correct